### PR TITLE
Added hurt sounds for brute and burn damage

### PR DESCRIPTION
**Pull request by JohnGinnane**
:cl: JohnGinnane
add: Added hurt sounds for brute and burn damage (cooldown 3 seconds long)
add: Added new sound for gibbing living people with lawnmower
/:cl:

Videos:
https://drive.google.com/file/d/1tqPMLlRrqCzFcObZunCc2iwzZ62YyIVo/view
https://drive.google.com/file/d/1gk3Kqze2tOuV06Y6B3w2lFJ45TI0z7OF/view
https://drive.google.com/file/d/1IiNdZjMQ4ltVQPEmeNlfaJK4bwvZi2DL/view

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
10/10 sound effects
0/10 execution
maybe make it a trait? havin this be for everyone sounds annoying
also there's a huge delay on sounds actually playin, why is that

**Comment by JohnGinnane** [comment]
Nah gonna leave it as is, I prefer it on everyone

Sounds are delayed so they play after hit sounds e.g. punch

**Comment by Maroonenahmoron** [comment]
There should be different sounds for each species

**Comment by bardofrage** [comment]
Sure, +1. Different sounds for different species would be cool though.

**Comment by Carbonhell** [comment]
Goyon is right,if the sound plays every single time it could be obnoxious. Either make it a toggleable preference, or make it play every 2 or 3 attacks.
+1 after one of the two solutions are done

**Comment by Jujumatic** [comment]
toggleable preference would most likely be better.

now we need to make something stupid that reduces the injury sound cooldown to 0 so we can go oof 900 times a second

File: code/game/sound.dm [hunk 1]
```diff
@@ -166,4 +166,9 @@
 				soundin = pick('sound/voice/beepsky/god.ogg', 'sound/voice/beepsky/iamthelaw.ogg', 'sound/voice/beepsky/secureday.ogg', 'sound/voice/beepsky/radio.ogg', 'sound/voice/beepsky/insult.ogg', 'sound/voice/beepsky/creep.ogg')
 			if("honkbot_e")
 				soundin = pick('sound/items/bikehorn.ogg', 'sound/items/AirHorn2.ogg', 'sound/misc/sadtrombone.ogg', 'sound/items/AirHorn.ogg', 'sound/effects/reee.ogg',  'sound/items/WEEOO1.ogg', 'sound/voice/beepsky/iamthelaw.ogg', 'sound/voice/beepsky/creep.ogg','sound/magic/Fireball.ogg' ,'sound/effects/pray.ogg', 'sound/voice/hiss1.ogg','sound/machines/buzz-sigh.ogg', 'sound/machines/ping.ogg', 'sound/weapons/flashbang.ogg', 'sound/weapons/bladeslice.ogg')
+			/* hippie start -- custom sounds */
+			if ("male_hurt")
+				var/num = rand(1, 21)
+				soundin = file("hippiestation/sound/voice/male_hurt" + num2text(num) + ".ogg")
```
Comment (Jujumatic) [inline code/game/sound.dm:172]: this is a very interesting way to select a number of files randomly, guess i learned something new today. cool!

File: hippiestation/code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -198,3 +201,25 @@
...
+		return
+
+	if (H.stat)
+		return
+
+	if (H.is_muzzled())
+		return
+
+	if (H.mind)
+		if (H.mind.miming)
```
Comment (Jujumatic) [inline hippiestation/code/modules/mob/living/carbon/human/species.dm:216]: maybe add like a 1% chance for the mime to go oof

with so much pain i don't know how mime would be able to keep quiet :D

File: hippiestation/code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -198,3 +201,25 @@
...
+			return
+
+	hurt_sound_cd = TRUE
+	addtimer(CALLBACK(src, .proc/play_hurt_sound, H), 5)
+
+/datum/species/proc/play_hurt_sound(mob/living/carbon/human/H)
+	// The sound frequency is set to 11,025 and I want to vary it by 25% both ways
+	playsound(H, "male_hurt", 75, 1, frequency = rand(11025*0.75, 11025*1.25))
```
Comment (Jujumatic) [inline hippiestation/code/modules/mob/living/carbon/human/species.dm]: guess i found out how to manually change frequencies too

**Review by Jujumatic — DISMISSED** [review]
nice code, i'll approve it as it looks fine but out of curiosity, why not just put queue_hurt_sound and play_hurt_sound together into one proc? as you're just making checks in queue_hurt_sound

**Comment by Jujumatic** [comment]
should have fixed conflict

**Review by Jujumatic — DISMISSED** [review]
go away bot my review is good

**Comment by JohnGinnane** [comment]
@Jujumatic I wanted a small delay between taking damage (which usually has its own sound effect like punching) and the hurt sound effect playing

I added a 3 second cooldown

**Review by Jujumatic — APPROVED** [review]
cool

**Comment by Jujumatic** [comment]
@k-k-karma you want anything else changed?

**Comment by Jujumatic** [comment]
+2, merging

**Pull request merged**